### DISCUSSION TITLE
Cant send the crazyflies commands after running another script with ROS

### DISCUSSION THREAD

**Discussion by Dr-Goodbeard** [USC-ACTLab/crazyswarm#760] [Q&A]
I currently have two crazyflies that I want to takeoff, figure out the closest landing zone, and then go and land on it. I have a separate script that I'm using to determine the closest landing zone; it's using the positions topics from vrpn client and ouputs each drone's assignment. 

The issue I'm having is that after takeoff I run this other script and then I can't send commands to the crazyflies. I get the error "service [/cf1/goTo] returned no response".

Any help would be appreciated.

**Comment by xkhainguyen** [comment]
I assume you created two identical CrazyflieServer nodes at once. I am not sure how to fix this in the best way either. 
What is the best way to control 2 crazyflies from 2 seperate scripts? @whoenig
